privacy/enforcement.go

```diff
@@ -9,11 +9,17 @@ type Enforcement struct {
 	GDPRGeo bool
 	GDPRID  bool
 	LMT     bool
+
+	// activities
+	UFPD       bool
+	Eids       bool
+	PreciseGeo bool
+	TID        bool
 }
 
 // Any returns true if at least one privacy policy requires enforcement.
 func (e Enforcement) Any() bool {
-	return e.CCPA || e.COPPA || e.GDPRGeo || e.GDPRID || e.LMT
+	return e.CCPA || e.COPPA || e.GDPRGeo || e.GDPRID || e.LMT || e.UFPD || e.PreciseGeo || e.Eids || e.TID
 }
```

Comment:
I recommend splitting `Any` into something like `AnyLegacy` and `AnyActivities`.

Reply:
Done